website/payments/api/v2/urls.py

```diff
@@ -5,12 +5,18 @@
     PaymentListView,
     PaymentDetailView,
     PayableDetailView,
+    PaymentUserCurrentView,
 )
 
 app_name = "payments"
 
 urlpatterns = [
     path("payments/", PaymentListView.as_view(), name="payments-list"),
+    path(
+        "payments/tpay-status/",
```

Comment:
I am not really sure this is the best way to do this. It's not an object, it's more an aggregation action and I would like those to be namespaced like `payments/actions/status`. I just have not had found a situation where this was really necessary so far. But maybe we can think of something better, since it could also  be `/payments/users/me`

Reply:
Shall we do `/payments/users/me` then?